client/components/checkbox/style.scss

```diff
@@ -0,0 +1,75 @@
+@import '../../scss/calypso-colors';
+@import '../../scss/rem';
+
+// ==========================================================================
+// Checkboxes
+// ==========================================================================
+
+.dops-checkbox.dops-checkbox {
+	display: inline-block;
+	box-sizing: border-box;
+	margin: 2px 0 0;
+	padding: 7px 14px;
+	width: 16px;
```

Comment:
We do. It's not doing anything there in any way so we could remove, but I thought we'd leave in case we wanted to rem-idy some of this stuff.